nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/test/java/org/apache/nifi/processors/kafka/KafkaPublisherTest.java

```diff
@@ -124,33 +112,9 @@ public void validateSuccessfulSendAsDelimited() throws Exception {
     }
 
     @Test
-    public void validateSuccessfulSendAsDelimited2() throws Exception {
```

Comment:
I see this test is just like the previous test, but is anything lost by not testing with a different delimiter?